### PR TITLE
done, sorry for vars names

### PR DESCRIPTION
**Pull request by anton-chornobai**

### CODE REVIEW

**Review by etojeDenys — CHANGES_REQUESTED** [review]
please, add Demo link to your PR

**Comment by anton-chornobai** [comment]
[DEMO LINK](https://anton-chornobai.github.io/js_task_fix_form_DOM/)

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,22 @@
 'use strict';
 
 // write code here
```
Comment (OleziO) [inline src/scripts/main.js:3]: Remove all comments from the code.

**Review by OleziO — CHANGES_REQUESTED** [review]
Make inputs enable, not disable.

**Comment by anton-chornobai** [comment]
demo -> https://anton-chornobai.github.io/js_task_fix_form_DOM/

**Review by mykhalenych — CHANGES_REQUESTED** [review]
Nice work, but change FIRSTNAME => FIRST NAME and LASTNAME => LAST NAME

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const inputs = [...document.querySelectorAll('.field-text')];
+
+const firstLetterCapitalizer = (s) => {
+  return s[0].toUpperCase() + s.slice(1).toLowerCase();
+};
+
+function camelCaseCheck(s) {
```
Comment (etojeDenys) [inline src/scripts/main.js:9]: Function names should be in camelCase. Rename `camelCaseCheck` to `checkCamelCase`.

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const inputs = [...document.querySelectorAll('.field-text')];
+
+const firstLetterCapitalizer = (s) => {
```
Comment (etojeDenys) [inline src/scripts/main.js:5]: Function names should be in camelCase. Rename `firstLetterCapitalizer` to `capitalizeFirstLetter`.

**Review by etojeDenys — CHANGES_REQUESTED** [review]
still not fixed 
image

File: src/scripts/main.js (hunk 3 above)
Comment (anton-chornobai) [reply src/scripts/main.js:5]: Sorry its probably old deploy

**Comment by anton-chornobai** [comment]
demo = https://anton-chornobai.github.io/js_task_fix_form_DOM/

File: src/scripts/main.js [hunk 4]
```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const inputs = [...document.querySelectorAll('.field-text')];
+
+const capitalizeFristLetter = (s) => {
+  return s[0].toUpperCase() + s.slice(1).toLowerCase();
+};
+
+function checkCamelCase(s) {
```
Comment (OleziO) [inline src/scripts/main.js:9]: Rename to `formatCamelCase`

**Review by OleziO — CHANGES_REQUESTED** [review]
Remove all unnecessary files from PR.

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]
Nice work, one minor mistake, in labels, words must also be separated by a space
image

**Pull request closed without merging**